Source/WebKit/Shared/haiku/WebEventFactory.h

```diff
@@ -0,0 +1,21 @@
+/*
+license
```

Comment:
there is a standard license for webkit, please use it in all files you create:

https://trac.webkit.org/export/242591/webkit/trunk/Source/WebCore/LICENSE-APPLE

of course replace copyright years and assign copyright either to yourself or to Haiku, inc, not to Apple.